host/xtest/regression_1000.c

```diff
@@ -835,12 +875,26 @@ static void xtest_tee_test_1009(ADBG_Case_t *c)
 static void xtest_tee_test_1010(ADBG_Case_t *c)
 {
 	unsigned n;
+	int idx;
+	uint32_t memref_sz[] = { 1024, 65536 };
 
 	for (n = 1; n <= 5; n++) {
 		Do_ADBG_BeginSubCase(c, "Invalid memory access %u", n);
 		xtest_tee_test_invalid_mem_access(c, n);
 		Do_ADBG_EndSubCase(c, "Invalid memory access %u", n);
 	}
+
+	for (idx = 0; idx < (sizeof(memref_sz) / sizeof(uint32_t)); idx++) {
```

Comment:
Please use `ARRAY_SIZE()` instead
Also here you're comparing a signed with an unsigned.
Any particular reason `idx` is an `int` while `n` is an `unsigned`?

Reply:
> `int` vs `unsigned`

No specific reason. I can use `unsigned` for both. Maybe `unsigned int` would be even better.

Reply:
Note: I use 2 size cases for the memref param: 1 below a page size and above quite far above page size. The latter is mainly to check the behaviour when the memref is a non physically contiguous registered shm buffer. In the later case, I even wonder the accuracy of the generated trace which shows the virtual base address, the physical base address and the size, but is it still relevant when the buffer is not contiguous.

My question here would rather be: is it useful to test several buffer size? even if it's harmless to test several.

Reply:
+1 to `unsigned int`, please fix `unsigned n` while you're at it.